tla-types/src/test/scala/at/forsyte/apalache/tla/typecomp/TestHybrid.scala

```diff
@@ -206,4 +206,7 @@ class TestHybrid extends BuilderTest {
     }
 
   }
+
+  // TODO: test hybrid FunOper methods (except/funDef)
+  // TODO: test shadowing ([x \in {x} |-> e] should fail)
```

Comment:
is it going to be added in a further PR, as you wrote in the PR comment?

Reply:
Yup